lib/pack_validate.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
-var beautify = require('js-beautify').js_beautify;
+var beautify;
```

Comment:
Default beautify to a function that returns the input unaltered so that you don't have to guard against it not being set elsewhere.